source/content/cms-admin.md

```diff
@@ -86,19 +86,21 @@ Do not update core using the Drupal Admin interface or Drush. Apply one-click up
    ![Drupal.org views module](../images/drupal-copy-module-link.png)
 
 
-    <Alert title="Note" type="info">
-
     Make sure the module has a version marked `8.x-*` before installing it on a Drupal 8 site.
-    </Alert>
 
 1. From the <span style="line-height:.9" class="glyphicons glyphicons-wrench"></span> **Dev** tab of your Pantheon Site Dashboard, click **Visit Development Site**. Sign in with your admin credentials.
 
 
-
-1. Under the **Manage** menu, click on **Extend**, then **+ Install new module**. Paste the URL from Step 1. You can also upload a tarball or zip file from your desktop:
+1. Under the **Manage** menu, click on **Extend**, then **+ Install new module**. Paste the URL from Step 1. You can also upload a `tarball` or `zip` file from your desktop:
 
     ![Installing a module on Drupal 8](../images/drupal-8-install-module.png)
 
+    <Alert title="Note" type="info">
+
+    There is a known bug in Drupal 8 impacting the **+ Install new module** and **+ Install new theme** buttons. See [Modules With Known Issues](/modules-known-issues#update-manager) for details.

```

Comment:
An argument could be made for utilizing the troubleshooting section of this doc instead of a callout. One advantage is having an anchor link specific to this issue to reference.

Reply:
Sounds good.